Tests/TABResourceLoaderTests/Mocks/MockObject.swift

```diff
@@ -23,6 +23,6 @@ extension MockObject {
 
 extension MockObject: Equatable {}
 
-func ==(lhs: MockObject, rhs: MockObject) -> Bool {
+func == (lhs: MockObject, rhs: MockObject) -> Bool {
```

Comment:
this is something that the default rules expect in swiftlint, I don't have preference either way to be honest